lnwallet/btcwallet/btcwallet.go

```diff
@@ -501,6 +502,7 @@ func minedTransactionsToDetails(
 			}
 
 			destAddresses = append(destAddresses, outAddresses...)
+			destAmounts = append(destAmounts, txOut.Value)
```

Comment:
I think the value must be repeated in order to match the number of `outAddresses` above. Should not happend very often for regular transactions, but it would be more correct IMO.